clueless/models.py

```diff
@@ -28,8 +42,9 @@ class Player(models.Model):
     Represents a player in the context of a clueless game.  Ties back to Django user
     """
     user = models.ForeignKey(User)
-    currentSpace = models.ForeignKey(Space)
-    currentGame = models.ForeignKey('Game') # game not defined yet, using string as lazy lookup
+    currentSpace = models.ForeignKey(Space, blank=True)
```

Comment:
Yeah definitely understand that rational just was wondering if leaving it null was a better way to handle it than putting in a filler value just because that leaves the ability for the value to accidentally be set to null later right?